lib/streamlit/elements/multiselect.py

```diff
@@ -44,7 +44,7 @@ def multiselect(
         ----------
         label : str
             A short label explaining to the user what this select widget is for.
-        options : list, tuple, numpy.ndarray, pandas.Series, or pandas.DataFrame
+        options : collection
             Labels for the select options. This will be cast to str internally
             by default. For pandas.DataFrame, the first column is selected.
```

Comment:
Is the comment accurate for the parameter now? I think the Pandas Dataframe will be needed for support, but perhaps I am not interpreting this correctly.

The original ask was here: https://github.com/streamlit/streamlit/issues/78 Tests make it pass, but I wasn't sure whether a dataframe was in fact a collection.

Reply:
The type is accurate, minus the possibility of someone passing a type that either has an index method that does something weird, or has a weird iterator definition that doesn't give the right things when turned into a list (like dataframes, which need special handling). 

The details of the parameter are also accurate, as we continue to have that special behavior for dataframes.